### ISSUE TITLE
Adding Avatars 3.0 Manager before unity has opened the project (using the creator companion) will cause a compile error

### ISSUE THREAD

**Issue by TheLastRar** [VRLabs/Avatars-3.0-Manager#32]
Testes on unity 2022.3.6

Create a new Avatar 2022 project using the Creator Companion
Add Avatar 3.0 Manager via the Creator Companion Manage Package Window
Click Open Project

Unity with show a pop-up stating compile errors, and asks you if you want to enter safe mode

If you enter safe mode (as suggested by unity) you will see this error `Packages\dev.vrlabs.av3manager\Editor\Elements\ClipsSwapAreaElement.cs(15,22): error CS0246: The type or namespace name 'ClipSwapItem' could not be found (are you missing a using directive or an assembly reference?`

**Comment by Cibbi** [comment]
it usually happens when multiple packages (in this case the sdk and the manager) are installed in a new project without opening it first, it's not an issue isolated to the manager as we did receive some reports of other packages doing the same, and we're not sure if we can do something on our end since it seems to be an issue with unity/vcc.

 Nonetheless we are keeping an eye on this issue since it does look fairly common and we would like to find a solution to it as well.

**Comment by Cibbi** [comment]
Update on the issue: #36 solves this issue, once the change is merged into main (with #37) we'll make a new release with the fix

**Comment by TheLastRar** [comment]
Nice! Thanks
